services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowCrudV2Spec.groovy

```diff
@@ -494,6 +495,37 @@ class FlowCrudV2Spec extends HealthCheckSpecification {
         ]
     }
 
+    def "Able to create a flow with vlan encapsulation type in lower case"() {
```

Comment:
Not the test case we want to see in 'functional-tests'. We are making specifications, i.e. describing/documenting correct behavior of our features or common use-case scenarios. This hardly can be called a documented feature or a real scenario. 
We are _not_ covering every small defect with explicit single-target tests, not here. Very narrow problem, this test is doomed to never catch any new defects, just increases the run time.

Option1: Make a real functional scenario that will reveal this exact problem. the test could have been named: `User can use 'get flow' response of existing flow to create a new flow`, which is an actual real-life scenario that will cover any other potential problems, not only flow encapsulation type lower case.
Problem with option1 is that it will only reveal issue when the v1->v2 mapping is on. Which is how the defect happens, which is the real problem. But mapping is hard to test, since it is a temporary thing, available only on staging. But the test still can be made and will catch the issue when run against the staging env

Option2 - just remove it, not worth automating (difficult due to need of mapping). Defect was successfully caught and raised without the autotest, so manual testing was enough.

Reply:
I choose option 2 :) 
What about second test with Bad request? Remove it too?

Reply:
Not sure. It's fine I guess. At least until we cover it at unit test level

Reply:
fixed